public/app/features/alerting/unified/mocks/alertmanagerApi.ts

```diff
@@ -18,6 +18,10 @@ export function mockAlertmanagerChoiceResponse(server: SetupServer, response: Al
   server.use(http.get('/api/v1/ngalert', () => HttpResponse.json(response)));
 }
 
+export function mockAlertmanagerChoiceResponseHandler(response: AlertmanagersChoiceResponse) {
```

Comment:
The method above is very similar - could we make that one use this generated handler?
So that one does something like `server.use(mockAlertmanagerChoiceResponseHandler(...));`

Reply:
yes!

Reply:
This handler is in `public/app/features/alerting/unified/mocks/server/handlers.ts` now, in `main` - so could use that one instead in this PR